spec/features/02_visitor_sees_show_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+feature "visitor sees show" do
+  scenario "details are on the page" do
+
+    oranges = Fruit.create(name: "oranges", description: "the color orange", sci_name: "Citrus × sinensis")
+
+    visit fruit_path(oranges.id)
+
+    expect(page).to have_content("oranges")
+    expect(page).to have_content oranges.name
+    expect(page).to have_content("the color orange")
+    expect(page).to have_content oranges.sci_name
```

Comment:
you are double testing here. you are testing that the name is there twice, and then that the description is there twice. 

Please make sure you are reviewing these PRs before submitting!

Reply:
corrected this test and added more depth to line 3